source/changelogs/core-2025.2.markdown

```
@@ -1358,6 +1358,260 @@ For a summary in a more readable format:
+- Handle previously migrated HEOS device identifier ([@andrewsayre] - [#137596])
+- Bump `aioshelly` to version `12.4.1` ([@bieniu] - [#137598])
+- Bump electrickiwi-api to 0.9.13 ([@mikey0000] - [#137601])
+- Bump ZHA to 0.0.48 ([@TheJulianJES] - [#137610])
+- Bump Electrickiwi-api to 0.9.14 ([@mikey0000] - [#137614])
+- Update google-nest-sdm to 7.1.3 ([@allenporter] - [#137625])
+- Don't use the current temperature from Shelly BLU TRV as a state for External Temperature number entity ([@bieniu] - [#137658])
+- Fix LG webOS TV turn off when device is already off ([@thecode] - [#137675])
+
+[#135231]: https://github.com/home-assistant/core/pull/135231
+[#137371]: https://github.com/home-assistant/core/pull/137371
+[#137448]: https://github.com/home-assistant/core/pull/137448
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "#137448"

(MD053, link-image-reference-definitions)